conductor/lib/nuke_utils.py

```diff
@@ -84,6 +84,33 @@ def get_plugins_info():
             plugins_info.append(PluginClass.get())
     return plugins_info
 
+def scrape_ocio_dependencies():
+    '''
+    Find the ocio config file (if it exists) and scrape it for external file dependencies
+    '''
+    config_filepath = get_ocio_config_filepath()
+
+    if config_filepath:
+        paths = ocio_utils.parse_ocio_config_paths(config_filepath)
+        return paths
+    
+    else:
+        return []
+
+def get_ocio_config_filepath():
+    '''
+    Return the OCIO config filepath from Nuke's root node.
+
+    Only return the filepath if color managment is set to use a custom OCIO
+    config.
+    '''
+    
+    ocio_config_path = None
+
+    if nuke.toNode("root").knobs()["OCIO_config"].value() == "custom":
```

Comment:
NIT: A stylistic thing I read somewhere - "Try to get out of the function early." 
If applied here, you would test for `not custom` and return there. Then you don't need to indent long lines. I think it's also a good habit when you have several tests: 
```
if not a:
   return None
if b:
   return None
return something
```

Reply:
That's true - I think it makes a lot of sense where the rest of the function has a lot more to it, though I'm also not a fan of multiple return statements and I think you have to choose which to do it based on the specific case.